docs/team.markdown

```diff
@@ -8,3 +8,19 @@ pagetitle: Localization Team
 ---
 
 Here you can learn more about the staff working in the Localization team at Mozilla.
+
+## Team Members
+
+| Name                       | Role                                                                        | Email                      |
+|----------------------------|-----------------------------------------------------------------------------|----------------------------|
+| Ayanaa Rahman              | Engineering intern (Pontoon)                                                | arahman@mozilla.com        |
+| Bryan Olsson               | Program manager: Firefox and Mozilla accounts                               | bolsson@mozilla.com        |
+| Delphine Lebédel           | Program manager: mobile                                                     | dlebedel@mozilla.com       |
+| Eemeli Aro                 | Engineer: MessageFormat 2.0                                                 | earo@mozilla.com           |
+| Francesco Lodolo (aka flod)| Team manager                                                                | flodolo@mozilla.com        |
+| Harmit Goswami             | Engineering intern (Pontoon)                                                | hgoswami@mozilla.com       |
+| Matjaž Horvat              | Engineer: main Pontoon developer                                            | mhorvat@mozilla.com        |
+| Peiying Mo                 | Program manager: Web projects and translation vendor management             | pmo@mozilla.com            |
+
+## Contact Us
+For more general inquiries, please reach out to [our the team email alias](mailto:l10n@mozilla.com).
```

Comment:
Remove "the" in `our the team email alias`. Suggest to change to: `For general inquiries, please contact us by using our team email alias.`